### ISSUE TITLE
Support for Discord instead of slack

### ISSUE THREAD

**Issue by jfrolich** [papercups-io/papercups#209]
We just moved to Discord. Would be great to have this support although not sure if it would work because discord doesn't have threading.

**Comment by reichert621** [comment]
hmm, do you think it would be viable to create a new channel per conversation, or would that get out of hand?

**Comment by jfrolich** [comment]
Yeah perhaps that would work. If it's easy to close/delete/archive channels of conversations that have completed.

**Comment by reichert621** [comment]
yeah definitely 👍 we'll look into it!

**Comment by dwwoelfel** [comment]
Another potential user for Discord: https://github.com/open-sauced/open-sauced/discussions/682

**Comment by LoanDEV** [comment]
+1

**Comment by afeiszli** [comment]
Is this still under consideration?
